azure-steps-windows.yml

```
@@ -73,6 +74,7 @@ steps:
     {
         $env:PATH = "$env:RTOOLS40_HOME\\mingw64\\bin;$env:PATH"
     }
+    ls $env:
```

```suggestion
    env | sort
```
Random try, to see if it prints out the full env for debugging.